api/src/main/scala/org/ensime/api/incoming.scala

```diff
@@ -53,7 +53,7 @@ final case class TypecheckFileReq(fileInfo: SourceFileInfo) extends RpcAnalyserR
 /**
  * Response with a `VoidResponse`.
  */
-final case class UnloadModuleReq(module: String) extends RpcAnalyserRequest
+final case class UnloadModuleReq(moduleId: EnsimeProjectId) extends RpcAnalyserRequest
```

Comment:
actually, I've changed my mind... let's just change the protocol message. @ktonga isn't this used in vim? This message will change to require a richer message.

In that case, bump the `1.9.1` to `1.9.2` in `outgoing.scala`